lib/apiservers/engine/backends/cache/image_cache.go

```diff
@@ -183,10 +182,10 @@ func (ic *ICache) Get(idOrRef string) (*metadata.ImageConfig, error) {
 	return copyImageConfig(config), nil
 }
 
-func (ic *ICache) getImageByDigest(digest digest.Digest) *metadata.ImageConfig {
+func (ic *ICache) getImageByDigest(digest string) *metadata.ImageConfig {
```

Comment:
I cannot remember why I did this.  I remember it was to get around a build issue, but that was before 1.13 officially got released.  With the updated vendored files, the old signature works so I reverted it back.